google-cloud-graalvm-samples/pom.xml

```diff
@@ -14,6 +14,7 @@
 
   <modules>
     <module>google-cloud-graalvm-pubsub-sample</module>
+    <module>trace-sample</module>
```

Comment:
Seems like a major departure from the naming convention.
I would expect `google-cloud-graalvm-trace-sample`.

Reply:
Actually I was going to rename the other samples to `pubsub-sample` (adopt this shorter naming convention) if that is OK.

I was just thinking that once we introduce `spring-cloud-gcp` in here, we'd have sample names something along the lines of `google-cloud-graalvm-spring-cloud-gcp-pubsub-sample` (perhaps we could cut some words out there...). But still, I was thinking we should adopt a shorter naming convention.

Reply:
Sure, makes sense!